v5/pivotal/projects.go

```diff
@@ -116,3 +116,39 @@ func (service *ProjectService) Get(projectId int) (*Project, *http.Response, err
 
 	return &project, resp, err
 }
+
+func (service *ProjectService) AddStory(
```

Comment:
I was just following what I saw with `AddTask` and `AddComment` in `stories.go`. Before I make the change I just want to be sure that it should be different than those?